mappings/net/minecraft/network/packet/client/CPacketWorldEvent.mapping

```diff
@@ -1,21 +1,21 @@
-CLASS none/gw net/minecraft/network/packet/client/CPacketEffect
-	FIELD a effectId I
+CLASS none/gw net/minecraft/network/packet/client/CPacketWorldEvent
+	FIELD a eventId I
```

Comment:
Why not just `id`? Wouldn't it be implied to be an `eventId` since this a `CPacketWorldEvent` class?